java-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -0,0 +1,363 @@
+= Query the database
+
+Once you have xref:connect.adoc[connected to the database], you can run queries using <<Cypher>> and the method `Driver.executableQuery()`.
+
+[TIP]
+`Driver.executableQuery()` was introduced with the version 5.8 of the driver. +
+For queries with earlier versions, see xref:transactions.adoc[Run your own transactions].
+
+
+== Write to the database
+
+To create a node representing a person named `Alice`, use the Cypher clause link:{neo4j-docs-base-uri}/cypher-manual/current/clauses/merge/[`MERGE`]:
+
+.Create a node representing a person named `Alice`
+[source, java]
+----
+// import org.neo4j.driver.QueryConfig;
+
+var result = driver.executableQuery("MERGE (:Person {name: $name})")  // <1>
+    .withParameters(Map.of("name", "Alice"))  // <2>
```

Comment:
True, I should also import `java.util.Map`. My intention is to keep examples short and to the point, to avoid cluttering each one with all the boilerplate of class and full imports, but showing "partial/new" imports as comments. If people need a full skeleton to work with, the `Full example` at the bottom provides it.